app/models/preprint.ts

```diff
@@ -59,6 +59,7 @@ export default class PreprintModel extends OsfModel {
     @attr('string') hasDataLinks!: PreprintDataLinksEnum;
     @attr('string') hasPreregLinks!: PreprintPreregLinksEnum;
     @attr('string') conflictOfInterestStatement!: string | null;
+    @attr('string') publicationCitation!: string;
```

Comment:
Has this new field been implemented on the backend already? Is it too late to rename this `originalPublicationCitation`, as this name makes it feel like the field is this preprint's citation info. Could also lump this property next to `originalPublicationDate` on line 44 as well